docs/src/makie.md

```diff
@@ -0,0 +1,79 @@
+# Makie.jl plots
+
+This page shows an example of plotting MCMCChains.jl with Makie.jl.
+The example is meant to provide an useful basis to build upon.
+Let's define some random chain and load the required packages:
+
+```@example makie
+using CairoMakie
+using DataFrames
+using MCMCChains
+
+chns = Chains(randn(300, 5, 3), [:A, :B, :C, :D, :E])
+```
+
+A basic way to visualize the chains is to show the drawn samples at each iteration.
+Colors depict different chains:
+
+```@example makie
+params = names(chns, :parameters)
+
+df = DataFrame(chns)
+n_chains = length(unique(df.chain))
+n_samples = nrow(df) / n_chains
+
+# Alternatively, use `CategoricalArrays.categorical`.
+df[!, :chain] = string.(df.chain)
+
+fig = Figure(; resolution=(1_000, 800))
+
+# Create and store separate axes for showing iterations.
```

Comment:
Since the intermediate figure is not plotted it doesn't seem necessary to save the axes?

Reply:
Yes, to tweak the appearance afterwards. For example, to drop all xticks for all plots **except** the lowest one, saving allows to write `hidexdecorations!.(values_axs[1:end-1]; grid=false)`. When moving this inside the loop, that would require a check to see whether we're not in the last iteration.

Also, to add a separate Legend to the figure, having stored the axes is required.

Reply:
> When moving this inside the loop, that would require a check to see whether we're not in the last iteration.

I know, I think it would be more natural to do this in the loop since we only plot the figure after hiding the decorations but are not interested in any intermediate figures.

> Also, to add a separate Legend to the figure, having stored the axes is required.

You can obtain them, I think with `getcontent` or `getcontents`. Alternatively, I think you could replace the for loop with a `map` that returns the axes (without decorations).

Reply:
> I know, I think it would be more natural to do this in the loop since we only plot the figure after hiding the decorations but are not interested in any intermediate figures.

I don't understand this point. We're also not showing things at that in between point.

> You can obtain them, I think with getcontent or getcontents. Alternatively, I think you could replace the for loop with a map that returns the axes (without decorations).

I've considered the map and do syntax and know that you're a fan of it, but I find that it's hard to read for newish Julia users. That's why I'm using a list comprehension and a for loop. Both are much more familiar to most readers.

Reply:
Can we agree to disagree on this one and merge as is?

Reply:
Wait yes only the loop might be the easiest to read. I'm changing it now